draft-irtf-cfrg-vdaf.md

```diff
@@ -2167,14 +2168,36 @@ post-processing.
 We remark that, taken together, these three functionalities correspond roughly
 to the notion of "Affine-aggregatable encodings (AFEs)" from {{CGB17}}.
 
+### Multiple proofs {#multiproofs}
+
+To improve soundness, the prover can construct multiple unique proofs for
+its measurement such that the verifier will only accept the measurement once
+all proofs have been thoroughly verified. Notably, several proofs using smaller
+field can offer the same level of soundness as a single proof using large field.
+
+To generate these proofs for a specific measurement, the prover can call
+`Flp.prove`, using different sets of prover/joint randomnesses. Then,
+the shares of these proofs are distributed to the verifier. Subsequently,
+ the verifier utilizes `Flp.query` to compute one share of verifier message
+ for each proof share, based on the measurement, the proof share, the joint
+ randomness, and unique query randomness. Finally, the
+verifier runs `Flp.decide` on each verifier message to make a decision. The
+validity of the measurement is determined through the logical conjunction
+of all Boolean outputs yielded by `Flp.decide`.
+
+
 ## Construction {#prio3-construction}
 
 This section specifies `Prio3`, an implementation of the `Vdaf` interface
 ({{vdaf}}). It has two generic parameters: an `Flp` ({{flp}}) and a `Xof`
-({{xof}}). The associated constants and types required by the `Vdaf` interface
-are defined in {{prio3-param}}. The methods required for sharding, preparation,
-aggregation, and unsharding are described in the remaining subsections. These
-methods refer to constants enumerated in {{prio3-const}}.
+({{xof}}). Moreover, an associated constant, `PROOFS`, can be defined for
+ each instance ({{prio3-instantiations}}). `PROOFS`, with a value within
+ the range of `[1, 256)`, denotes the number of proofs ({{multiproofs}})
+ and is set to `1` by default, unless explicitly specified. The associated
```

Comment:
I hightlight the default value `1` here. Maybe we don't have to repeat it in each instance?

Reply:
Works for me.